samples/net/ocpp/src/credentials/ec.crt

```diff
@@ -0,0 +1,10 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
I think we didn't want to include pregenerated credentials (the whole point of `USE_DUMMY_CREDS` config)?